internal/events/events_test_utils.go

```diff
@@ -0,0 +1,84 @@
+package events
```

Comment:
Don't block merge on this, but do we have a convention for test utils?
I don't particularly like adding too much to the "real" package just for test utils.
Especially not exported functions and types.

Not sure who to ask about this, @filanov maybe?